test/schema.test.ts

```diff
@@ -460,3 +458,9 @@ describe("tsconfig.json", () => {
         }
     });
 });
+
+describe("Required", () => {
+    // this part is needed to resolve ts script internaly
+    require("ts-node/register");
```

Comment:
Hmm, does this mean I cannot call `typescript-json-schema` with just node (for example with npx)?